runtests.sh

```diff
@@ -23,19 +32,27 @@ export PYTHONPATH=`pwd`/src
 echo "Create Virtualenv for Python deps ..."
 function prepare_venv() {
     VIRTUALENV=`which virtualenv`
-    if [ $? -eq 1 ]; then
+    if [ $? -eq 1 ]
+    then
         # python34 which is in CentOS does not have virtualenv binary
         VIRTUALENV=`which virtualenv-3`
     fi
 
     ${VIRTUALENV} -p python3 venv && source venv/bin/activate
+    if [ $? -ne 0 ]
+    then
+        printf "${RED}Python virtual environment can't be initialized${NORMAL}"
+        exit 1
+    fi
     pip install -U pip
     python3 `which pip3` install -r requirements.txt
 
 }
 
 [ "$NOVENV" == "1" ] || prepare_venv || exit 1
 
+# now we are surely in the Python virtual environment
+
 `which pip3` install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@561636c
```

Comment:
If we are good with running `pip` here than let's change it to `pip` from `which pip3`. WDYT @tisnik ?